runtime/binding-http/src/test/java/io/aklivity/zilla/runtime/binding/http/internal/streams/rfc7230/server/ConnectionManagementIT.java

```diff
@@ -69,6 +69,16 @@ public void shouldSendRequestWithHeaderOverride() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Configuration("server.override.with.yaml")
+    @Specification({
+        "${net}/request.with.header.override.with/client",
+        "${app}/request.with.header.override.with/server" })
+    public void shouldSendRequestWithHeaderOverrideWith() throws Exception
```

Comment:
```suggestion
        "${net}/request.with.route.header.overrides/client",
        "${app}/request.with.route.header.overrides/server" })
    public void shouldSendRequestWithRouteHeaderOverrides() throws Exception
```

Same naming feedback for other similar scenarios, avoids double-with.